### ISSUE TITLE
Make fungi compostable

### ISSUE THREAD

**Issue by thcrt** [AstralOrdana/Spelunkery#183]
Fungi such as the Conk Fungus should be compostable. This would fit in line with the compostability of vanilla mushrooms [[source]](https://minecraft.wiki/w/Composter#Composting) and give another use to some items if they otherwise wouldn't be needed.

**Comment by thcrt** [comment]
For the time being, I'm using this KubeJS server script:

```js
ServerEvents.compostableRecipes(event => {
    // Make Spelunkery fungi compostable
    // Vanilla mushrooms are already compostable with a chance of 65%,
    // so we replicate that here:
    event.add("spelunkery:button_mushroom", 0.65);
    event.add("spelunkery:crimini", 0.65);
    event.add("spelunkery:portabella", 0.65);
    event.add("spelunkery:conk_fungus", 0.65);
    event.add("spelunkery:inkcap_mushroom", 0.65);
    event.add("spelunkery:white_inkcap_mushroom", 0.65);
    event.add("spelunkery:phosphor_fungus", 0.65);
    event.add("spelunkery:mushgloom", 0.65);
    event.add("spelunkery:milly_bubcap", 0.65);
    event.add("spelunkery:sporophyte", 0.65);
    event.add("spelunkery:tall_sporophyte", 0.65);
    event.add("spelunkery:tangle_roots", 0.65);
    // Grilled Portabella gets a chance of 85% in line with Baked Potatoes
    event.add("spelunkery:grilled_portabella", 0.85);
});
```
Just adding it here in case it helps anyone else who comes across this.

**Comment by AstralOrdana** [comment]
implemented for 0.3.10
